edge-modules/Relayer/Program.cs

```diff
@@ -43,6 +44,11 @@ static async Task Main(string[] args)
             {
                 Logger.LogError(ex, "Error occurred during Relayer.");
             }
+            finally
+            {
+                moduleClient?.CloseAsync();
+                moduleClient?.Dispose();
```

Comment:
Why do we need to dispose?

Reply:
Just to clean up moduleClient as it implements IDisposable interface.